INSTALL.md

```diff
@@ -0,0 +1,458 @@
+# Installation Guide
+
+This document describes steps necessary to install the CEL Admission Webhook Shim
+on your local cluister
+
+# Easy Installation
+
+## For Staging/Production using Kpt
+
+> 🚧 WORK IN PROGRESS 🚧
+
+## For Testing using Kind
+
+This method uses a local [`kind`](https://kind.sigs.k8s.io) cluster and is recommended only for testing/development. 
+
+Run the following script to build the image from source, and generate a minimum working webhook configuration:
+
+```sh
+cd $REPO_PATH
+# Build container image to run the controller
+docker build -t kubernetes-x/cel-shim-webhook:v1 ./
+# Make container image available to Pods in the Kind cluster
+# NOTE: If your kind cluster is named, this will also need the --name argument with the name of your cluster
+kind load docker-image kubernetes-x/cel-shim-webhook:v1
+# Generate deployment, serviceaccount, certificates, etc into ./_output/manifests
+./hack/gen-manifests.sh
+# Apply required manifests for webhook
+kubectl apply -f ./_output/manifests --server-side=true
+```
+
+This script creates a self-signed CA and a server certificate, and outputs manifests containing webhook configurations and TLS secrets.
+
+After running the final `kubectl apply` the CEL Admission Webhook should
+be active on your kind cluster and ready to enforce policies.
+
+
+# Manual Installation
+
+> 🚧 WORK IN PROGRESS 🚧
+
+Requirements:
+  1. Kubernetes 1.x+ (TODO: determine minimum k8s version for webhook)
```

Comment:
Webhook is beta since 1.9 and become stable in 1.16

Reply:
Since CRD is stable in 1.16 as well, how about to set the requirement to 1.16?

Reply:
added